lib/fab_circular_menu.dart

```diff
@@ -146,19 +160,21 @@ class _FabCircularMenuState extends State<FabCircularMenu>
           padding: widget.fabMargin,
           child: FloatingActionButton(
             child: open ? widget.fabCloseIcon : widget.fabOpenIcon,
-            backgroundColor: fabColor,
+            backgroundColor: open ? fabOpenColor : fabCloseColor,
             onPressed: () {
               if (!animating && !open) {
                 animating = true;
                 open = true;
                 animationController.forward().then((_) {
                   animating = false;
+                  widget.onDisplayChange({'isOpen': open});
```

Comment:
If the user has not set a callback function, `_defaultVoidFunc` will be called and will throw an error because of wrong number of arguments (`_defaultVoidFunc` receives no arguments)